packages/data/src/components/use-select/test/index.js

```diff
@@ -132,5 +132,93 @@ describe( 'useSelect', () => {
 			children: 'bar',
 		} );
 	} );
+	describe( 'rerenders as expected with various mapSelect return ' +
+		'types', () => {
+		const getComponent = ( mapSelectSpy, dependencyKey ) => ( props ) => {
+			const dependencies = props[ dependencyKey ];
+			mapSelectSpy.mockImplementation(
+				( select ) => select( 'testStore' ).testSelector( props.keyName )
```

Comment:
The parameter in the selector is unused.

Reply:
We can remove all the usages of `dependencyKey` and `keyName` here. The test never changes the dependency so it would be just like passing an empty array instead.

Reply:
see f3fe2e0